tools/gen-magic-strings.py

```diff
@@ -161,10 +167,10 @@ def calculate_magic_string_guards(defs, uses, debug=False):
     extended_defs = []
 
     for str_ref, str_value in defs:
-        if str_ref not in uses:
-            print('warning: unused magic string {str_ref}'
-                  .format(str_ref=str_ref))
-            continue
+        # if str_ref not in uses:
+        #     print('warning: unused magic string {str_ref}'
+        #           .format(str_ref=str_ref))
+        #     continue
```

Comment:
Is it ok to have this commented out? Shouldn't we keep this?

Reply:
Yes we should keep it. I have left it commented accidentally.